src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -119,6 +119,10 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_query_acc_atomic_support(MPI_Datatype dt
     if (winattr & MPIDI_WINATTR_ACCU_SAME_OP_NO_OP)
         *count = (size_t) MPIDI_OFI_WIN(win).acc_hint->dtypes_max_count[dt_index];
 
+    MPL_DBG_MSG_FMT(MPIDI_CH4_DBG_GENERAL, VERBOSE,
+                    (MPL_DBG_FDEST, "Query atomics support: max_count 0x%lx, dtsize 0x%lx", *count,
+                     *dtsize));
+
```

Comment:
I am against permanently adding too much debug  messages. It is relatively simple to manually insert the log or printf during debugging when needed. The example here will add a log entry for every `acc` atomic operations, right? I think it can easily drown out other debug messages if an application is atomic-heavy.

Reply:
It generates this line only when log is on. For each `acc` path, it already generates many lines to track the calling stack (e.g., MPIR_FUNC_VERBOSE_ENTER, MPIR_FUNC_VERBOSE_EXIT). Thus, I don't think adding this message will pollute log. But it provides very useful info for us to figure out the cause why it does not use native atomics (at least it takes me long time to figure it out when testing on a new platform).

The log functionality will not be needed if we always manually add such prints only at debugging time :-)

Reply:
> The log functionality will not be needed if we always manually add such prints only at debugging time :-)

That is my opinion. :) I think we should delete all debug logging from repository unless those logging output makes sense to a non-mpich user. 

The log functionality is needed so we can manually add it (printf sometime is too slow or too interfering).  But with permanent embedded debug output , when we do need to debug, we'll have to parse the huge log, and potentially getting confused by unrelated output. In theory, the class can be used to filter output. But they are just added complexity. The time I needed to understand what class to filter and how to filter, I could just manually insert my desired debug output and be done with it.

Reply:
I do believe the log can be useful. One example is that, a user reported to me that he cannot get native atomics on his platform for unknown reason. Then I can easily figure out the cause if the user can send me the log. 

Anyway, this is really personally preference. I hope you will not block this PR because of the log preference:-)